swctl/main.go

```diff
@@ -60,12 +61,14 @@ func main() {
 		altsrc.NewStringFlag(cli.StringFlag{
 			Name:     "display",
 			Required: false,
-			Usage:    "display `style` of the result, supported styles are: json, yaml",
+			Usage:    "display `style` of the result, supported styles are: json, yaml, table",
+			Value:    "yaml",
```

Comment:
Good catch, but I think the default should be JSON, it's more useful in to pipe commands

Reply:
done